Norm_end_to_end.Rmd

```diff
@@ -218,41 +283,31 @@ samples.
 
 ```{r}
 #' Compute median and generate normalization factor
-mdns <- apply(assay(res, "raw_filled"), MARGIN = 2, ## should be done on raw ? 
+mdns <- apply(assay(res, "raw_filled"), MARGIN = 2,
```

Comment:
no, the median for samples with a high number of filled-in signals (i.e. fewer detected signals) tends to be high, so we would overestimate the mean abundance and scaling by that would lower all intensities in that sample